content/docs/overview.md

```diff
@@ -1262,6 +1277,11 @@ Slices and dynamic arrays can be explicitly allocated with the built-in `make` p
 a := make([]int, 6)           // len(a) == 6
 b := make([dynamic]int, 6)    // len(b) == 6, cap(b) == 6
 c := make([dynamic]int, 0, 6) // len(c) == 0, cap(c) == 6
+d := []int{1, 2, 3}           // a slice literal, for comparison
+
+// with an explicit allocator:
+e := make([]int, 6, context.allocator)
+f := make([dynamic]int, 0, 6, context.temp_allocator)
```

Comment:
I'd just show context.allocator here to minimize confusion.

Reply:
Updated. Something I didn't realize while writing the original is that slices *don't* remember their allocator. I.e., this segfaults:

```odin
package main

import "core:fmt"

main :: proc() {
	a := make([]int, 5, context.temp_allocator)
	delete(a)
}
```

But it wouldn't with
```
	a := make([dynamic]int, 5, 5, context.temp_allocator)
```

nor does this:

```odin
package main

import "core:fmt"

main :: proc() {
	context.allocator = context.temp_allocator
	a := make([]int, 5)
	delete(a)
}
```